src/signet.cpp

```diff
@@ -65,8 +65,9 @@ static uint256 ComputeModifiedMerkleRoot(const CMutableTransaction& cb, const CB
     return ComputeMerkleRoot(std::move(leaves));
 }
 
-SignetTxs SignetTxs::Create(const CBlock& block, const CScript& challenge)
+Optional<SignetTxs> SignetTxs::Create(const CBlock& block, const CScript& challenge)
 {
+    const auto invalid = []() -> Optional<SignetTxs> { return nullopt; };
```

Comment:
Should just replace `return invalid()` with `return nullopt` directly.

Reply:
thx done